.env

```diff
@@ -0,0 +1 @@
+MONGODB_URI=mongodb+srv://Alice:orRfmHt03jqfpbCg@vr.lkf3l.mongodb.net/VisitRosenheim?retryWrites=true&w=majority
```

Comment:
I think this should not be part of git, right?

Reply:
Hi Paul, thx for your review, you are absolutely right. I removed the file and ignored it retroactively. 👍